data/json/items/resources/metal.json

```diff
@@ -271,6 +271,44 @@
     "color": "light_gray",
     "ammo_type": "components",
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+    {
+    "id": "orichalcum_lump",
```

Comment:
I do not think it belongs to vanilla and you need to move it to corresponding mod.